packages/property-provider/src/CredentialsProviderError.spec.ts

```diff
@@ -11,8 +11,7 @@ describe(CredentialsProviderError.name, () => {
     });
 
     it("when created using from()", () => {
-      // ToDo: set instanceof to be CredentialsProviderError
-      expect(CredentialsProviderError.from(new Error("PANIC"))).toBeInstanceOf(Error);
+      expect(CredentialsProviderError.from(new Error("PANIC"))).toBeInstanceOf(CredentialsProviderError);
```

Comment:
Add a test case for `toBeInstanceOf(Error)`?

Reply:
Added in [`84d4cc4` (#3440)](https://github.com/aws/aws-sdk-js-v3/pull/3440/commits/84d4cc43be1e08d422bc5b6d1b34155b029428e6)